src/screens/ContactScreen/elements/Search.js

```diff
@@ -95,7 +97,7 @@ const styles = StyleSheet.create({
   },
   wrapperSecondaryIcon: {
     position: 'absolute',
-    right: 8,
+    right: 12,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `right` value in the `wrapperSecondaryIcon` style has been updated. This will affect the positioning of the secondary icon. Ensure this change aligns with the intended design.
```diff
- right: 8,
+ right: 12,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->